src/bit_str.jl

```diff
@@ -183,43 +166,42 @@ function parse_bit(::Type{T}, str::String) where {T <: Integer}
         else
             error("expect 0 or 1, got $each at $k-th bit")
         end
+        (isbitstype(T) && k>bsizeof(T)) && error("Bit String too long! use Long bit string?")
```

Comment:
the original error msg didn't indicate anything specific. error msg should point to a specific API name.

```suggestion
        (isbitstype(T) && k>bsizeof(T)) && error("string length is larger than $(bsizeof(T)), use @lbit_str instead")
```

Reply:
When it errors, traceback information points to the API.